specs/ws-api.yaml

```diff
@@ -281,12 +281,12 @@ components:
           const: update
         topic:
           type: string
-          enum: ['newUserOperations']
+          enum: ['newSolverInputs']
         data:
           type: object
           properties:
             userOperation:
-              $ref: '#/components/schemas/UserOperation'
+              $ref: '#/components/schemas/SolverInput'
```

Comment:
The schema is invalid, try pasting this file content into https://studio.asyncapi.com/ to get errors. `SolverInput` in not defined.

Reply:
done